app.rb

```diff
@@ -0,0 +1,66 @@
+class App
+  def initialize
+    # Initialize any required resources here
+  end
+
+  def run
+    welcome_message
+
+    loop do
+      display_menu
+      choice = get_user_choice
+      break if choice == 7
+
```

Comment:
- Hey, when trying to run your app using the `ruby main.rb`, the app throws the error below. Please, kindly fix this.

 NOTE: The image may be broken, please, test your app again to see the error it throws. Here's the error:
 
 ```
/Desktop/OOP-school-library/app.rb:11:in `block in run': undefined local variable or method `get_user_choice' for #<App:0x00007f768f248aa8> (NameError)

      choice = get_user_choice
               ^^^^^^^^^^^^^^^
        from /home/zieeco/Desktop/OOP-school-library/app.rb:9:in `loop'
        from /home/zieeco/Desktop/OOP-school-library/app.rb:9:in `run'
        from main.rb:4:in `<main>'
```

![image](https://github.com/Bupilipili/OOP-school-library/assets/59397159/83a8eeda-48d8-4609-8c02-ca8a4822216e)